schemas/stsci.edu/asdf/transform/brokenpowerlaw1d-1.0.0.yaml

```diff
@@ -0,0 +1,44 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/asdf/transform/brokenpowerlaw1d-1.0.0"
```

Comment:
Shall we continue the proud tradition of snake case filenames and ids/tags?

Reply:
whoops forgot about that!